backend/Backend.cpp

```diff
@@ -125,6 +125,7 @@ bool Backend::IsClientLayer(HwcDisplay *display, HwcLayer *layer) {
          display->color_transform_hint() != HAL_COLOR_TRANSFORM_IDENTITY ||
          (layer->GetLayerData().pi.RequireScalingOrPhasing() &&
           display->GetHwc2()->GetResMan().ForcedScalingWithGpu()) ||
+	 !display->GetDeviceRectSupport(layer->GetLayerData().pi.display_frame);
```

Comment:
please double check the code, ; should be II ?

Reply:
If input source resolution < device min resolution, return to client compostion.